Issue/#89 svg tool review with big cities

agalbachicar opened the pull request:
Changes in the SVG tool have been made to detect possible duplicates of all the items. Now, if an error appears, parsing will be stopped and a message indicating the problem and the line will appear. Also, four sample files have been added under the samples/rndf/wrong directory.

closes #89

basicNew commented:
In general LGTM @agalbachicar, thanks! Just two minor things:

- Could we rename the RNDFs so they have a better description (like `duplicate_segments_ids`) or something along those lines? Otherwise is hard to understand why they are different.
- Can you make the smallest possible RNDF to reproduce the issue? It would be much clearer to depict e.g. the duplicate waypoint example with just 3 segments instead of 12 (if it is possible, of course).

agalbachicar commented:
@basicNew I have changed the RNDF sample files with issues, changed its name and removed one duplicate case (duplicate segment id). Also, I've added a new sample case of duplicate exit point in the same lane. I think now it's right.

basicNew commented:
Awesome, thanks a lot @agalbachicar !
